jenkins/stage-maven-release.jenkinsfile

```diff
@@ -12,38 +12,31 @@ pipeline {
             alwaysPull true
         }
     }
+    triggers {
+        GenericTrigger(
+            genericVariables: [
+                [key: 'ref', value: '$.ref'],
+                [key: 'VERSION', value: '$.ref', regexpFilter: 'refs/tags/v' ],
+            ],
+            tokenCredentialId: 'jenkins-opensearch-java-generic-webhook-token',
+            causeString: 'A tag was cut on opensearch-project/opensearch-java repository causing this workflow to run',
+            printContributedVariables: false,
+            printPostContent: false,
+            regexpFilterText: '$ref',
+            regexpFilterExpression: '^refs/tags/.*'
+        )
+    }
     environment {
-        VERSION = "${params.VERSION}"
-        ARTIFACT_PATH = "$WORKSPACE/build/repository/org/opensearch/client/opensearch-java/${VERSION}"
+        ARTIFACT_PATH = "$WORKSPACE/build/repository/org/opensearch/client/opensearch-java/$VERSION"
     }
     stages {
-        stage('parameters') {
-            steps {
-                script {
-                    properties([
-                        parameters([
-                            string(
-                                name: 'REF',
-                                trim: true
-                            ),
-                            string(
-                                name: 'VERSION',
-                                trim: true
-                            )
-                        ])
-                    ])
-                    if (params.REF.isEmpty() || params.VERSION.isEmpty()) {
-                        currentBuild.result = 'ABORTED'
-                        error('Missing REF and/or VERSION.')
-                    }
-                }
-            }
-        }
         stage('Publish to Maven Local') {
             steps {
                 // checkout the commit
-                git url: 'https://github.com/opensearch-project/opensearch-java.git', branch: 'main'
-                sh('git checkout ${REF}')
+                checkout([
+                    $class: 'GitSCM', userRemoteConfigs: [[url: 'https://github.com/opensearch-project/opensearch-java.git']],
+                    branches: [[name: "$ref"]]
+                        ])
```

Comment:
tabs vs. spaces strikes again!

Reply:
Its a tab! I promise :)

Reply:
I think that's why it looks weird. Make it all spaces! :) 

Feel free to merge at will anyway.

Reply:
Looks like GHA taking its sweet time to run workflows. Will do once everything is green. Thanks!

Reply:
It runs integ tests, you can just merge this since it doesn't touch any code.